### PR TITLE
[for 2.6.2] Fix/1503 close the sidebar when the device is a mobile

### PR DESCRIPTION
**Pull request by metzgegu**
#1503 

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [x] Tested by quality team

### CODE REVIEW

**Comment by metzgegu** [comment]
I can't add automated test because Cypress don't support mobile testing for now

**Comment by PhilippeAccorsi** [comment]
@metzgegu I'm not sure but I think in this link we have information to make mobile test with cypress https://medium.com/@sumn2u/testing-mobile-devices-using-cypress-99a1c1294057

**Comment by PhilippeAccorsi** [comment]
Tested and verified on branch with android (firefox, chrome) and IOS (safari).

**Comment by metzgegu** [comment]
> @metzgegu I'm not sure but I think in this link we have information to make mobile test with cypress https://medium.com/@sumn2u/testing-mobile-devices-using-cypress-99a1c1294057

After a discussion with @Skylsmoi we only need a unit test for this issue
Two Cypress configurations for mobile and browsers may be useful in the future, thanks for the links :+1:

File: frontend/test/src/container/Sidebar.spec.js [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
+      }
+    },
+    t: tradKey => tradKey
+  }
+
+  const ComponentWithHOC = withRouterMock(SidebarWithoutHOC)
+
+  const wrapper = shallow(
+    <ComponentWithHOC {...props} />
+  ).dive() // INFO - GM - 2020-03-04 - Need to dive here because we can't use mount (Dnd problems)
```
Comment (Skylsmoi) [inline frontend/test/src/container/Sidebar.spec.js]: please a bit more specific about the dnd issue.
Either some info or a link to a known issue

**Pull request merged**